app/src/main/java/io/appium/uiautomator2/handler/Flick.java

```diff
@@ -15,13 +31,14 @@
 import io.appium.uiautomator2.model.AppiumUIA2Driver;
 import io.appium.uiautomator2.model.Session;
 import io.appium.uiautomator2.utils.Logger;
-import io.appium.uiautomator2.utils.Point;
+import io.appium.uiautomator2.model.Point;
 import io.appium.uiautomator2.utils.PositionHelper;
 
 import static io.appium.uiautomator2.utils.Device.getUiDevice;
 import static io.appium.uiautomator2.utils.ModelUtils.toModel;
 
 public class Flick extends SafeRequestHandler {
+    private static final double SPEED_MAGIC = 1250.0;
```

Comment:
It's out of curiosity, do you remember the reason why we set this magic number?
(like wait time in XCUITest)

Reply:
no idea. This number was here from the very beginning

Reply:
I just moved it to a constant for better readability